build.psm1

```diff
@@ -1720,7 +1733,9 @@ function Start-PSBootstrap {
 
                 # Install dependencies
                 # ignore exitcode, because they may be already installed
-                Start-NativeExecution { brew install $Deps } -IgnoreExitcode
+                Start-NativeExecution {
+                    Invoke-Expression "$PackageManager install $Deps"
+                } -IgnoreExitcode
```

Comment:
Please remove the `Invoke-Expression`
```
                Start-NativeExecution {
                    &$PackageManager install $Deps
                } -IgnoreExitcode
```

Reply:
If that doesn't work
```
Start-NativeExecution ([ScriptBlock]::Create("$PackageManager install $Deps"))
```

Reply:
Are you sure about that? The comment above explicitly says to use invoke-expression for sudo interpolation to work properly.

Reply:
New line 1756 we are doing what I'm suggesting.  I'm 100% sure one of these will work.

Reply:
Pretty sure the `& $Packagemanager install @Deps` method would work just fine, if not better... all depends on the syntax expected by `brew` or `port` -- do they expect a space-delimited list of values, or a comma-delimited list of values for the package names?

Reply:
Both of them want a space-delimited list of package names. But the problem with & is not the package list but rather the fact that port needs sudo to run, and if the user specifies -NoSudo, the $sudo expands to an empty string, which works fine with Invoke-Expression or ScriptBlock, but fails when called with &

Reply:
Oh, fair enough I suppose. You could just if/else that though. :)

Reply:
@vexx32 We have many other places we are creating script blocks... I'm fine with this for now.